google/resource_container_cluster_test.go

```diff
@@ -199,6 +199,31 @@ func TestAccContainerCluster_withNodeConfig(t *testing.T) {
 	})
 }
 
+func TestAccContainerCluster_withNodeConfigNotSorted(t *testing.T) {
```

Comment:
This seems to imply that the node config itself is not sorted, but it's the oauth_scopes that we actually care about. Since there's already a test that's checking the full node config, I think an easier way to test this would be just to change the order of the oauth_scopes in TestAccContainerCluster_withNodeConfig